src/app/components/Header/components/BackButton/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { TouchableOpacity, Image } from 'react-native';
+
+import styles from './styles';
+
+interface Props {
+  onPress: () => void;
+  backButton: number;
```

Comment:
i think there is an ImageType from `react-native`. Check it out or leave it like this

Reply:
Alright, I'll look into it 👀